pennylane/transforms/draw.py

```diff
@@ -18,7 +18,8 @@
 """
 from functools import wraps
 
-import pennylane as qml
+from pennylane.drawer import tape_mpl
+from pennylane.wires import Wires

```

Comment:
How come this was changed?

Reply:
While double-checking import paths, I saw `qml` was only used twice. At that number it's cleaner and more performant to just import directly what we need.